src/ui/actions.ts

```diff
@@ -1,17 +1,20 @@
 import { MarkdownView, TFile, Vault, Workspace } from "obsidian";
 import EventCache from "src/core/EventCache";
+import { FullCalendarSettings } from "./settings";
 
 /**
  * Open a file in the editor to a given event.
  * @param cache
  * @param param1 App
  * @param id event ID
+ * @param settings settings from FullCalendar
  * @returns
  */
 export async function openFileForEvent(
     cache: EventCache,
     { workspace, vault }: { workspace: Workspace; vault: Vault },
-    id: string
+    id: string,
+    settings?: FullCalendarSettings
```

Comment:
```suggestion
    openInNewTab: boolean
```
Could you pass the setting in as its own parameters rather than passing in the whole settings object? I'd prefer it to be explicit what settings `openFileForEvent` is using from its type signature.

Reply:
Sure, just updated the branch